### DISCUSSION TITLE
GitHub Education Verification Process keeps rejecting with valid documents.

### DISCUSSION THREAD

**Discussion by illyrius666** [community/community#56747] [GitHub Education] [answered]
Great Github staff,

I am opening this discussion because I am sick and tired of trying to make the system able to read 2 same names. The one on my documents and the one on my GitHub profile. It's unbelievable that a company like this cannot manage to make a system that can find similarities between names. How hard is it to see that for example Bob Johnson = Bob Johnson? But somehow the system is unable to read that. It keeps giving me the rejection reason that I should change my profile to match the affiliated documents.
I tried to use the enrollment document, school portal, and receipts from the bursar. All of them are added to the firm fill by the direct camera. I have plenty of proof to showcase that I am a student at the university, but yeah apparently it's not possible to actually make it get processed.
The university where I currently follow my course doesn't issue school mail nor does it have school id cards. This has been outdated in the Netherlands for a while now.

I would love and appreciate it if a moderator or admin could help me with this problem.

Yours sincerely,
IllyriusQ

**Comment by illyrius666** [comment]
###suggestion

I have seen that some CAD software companies make use of systems that does the verification by accessing government sites. this is the ideal verification system since it cannot be falsified.

No stress and frustration with documents since the government site has everything registered already.

**Comment by jge162** [comment]
Hello @IllyriusQ Couple questions, are you using your school email as your main email on your GitHub account. Second try using transcripts. That’s how I was approved and many others I’ve helped here. Report back after you resubmit. Good luck.

**Reply by illyrius666** [reply]
"Are you using your school email as your main email on your GitHub account"
I wish in this case I had one, but as I noted in my first message the uni where I am enrolled doesn't issue school emails.

"Second try using transcript"
Would be an option if I got marks. but since I just started that's not generated yet. And yes the uni where I am at has courses that have 2 start dates per year.
Basically, I haven't had my first exam yet.

**Reply by jge162** [reply]
You should have semester courses showing what classes you are taking. Provide that document. 

School email is not needed.

**Reply by jge162** [reply]
Also your profile does not show your name. I just saw an L

**Reply by illyrius666** [reply]
> You should have semester courses showing what classes you are taking. Provide that document.
> 
> School email is not needed.

![image](https://github.com/community/community/assets/28700752/f5595e29-0fb9-4d1d-a841-090c25a76b08)

This one?

**Reply by illyrius666** [reply]
or this one?

![image](https://github.com/community/community/assets/28700752/7706c27f-9bd1-49da-b0fd-ec4ae85a3ca5)

**Reply by illyrius666** [reply]
btw its in dutch

**Reply by jge162** [reply]
Is that your current course you are taking? Looks like too many on the page. Usually someone takes 4-6 classes.

**Reply by illyrius666** [reply]
> Also your profile does not show your name. I just saw an L

![image](https://github.com/community/community/assets/28700752/c060e0fd-7e68-4ac2-8656-226b0cd1bc51)

Iam pretty sure that's my name. that's also how it is issued on the official documents.

**Reply by jge162** [reply]
> or this one?
> 
> ![image](https://user-images.githubusercontent.com/28700752/241936315-7706c27f-9bd1-49da-b0fd-ec4ae85a3ca5.png)

What is that. The road map to your degree? That does not look like the classes you are taking now.

**Reply by jge162** [reply]
> > Also your profile does not show your name. I just saw an L
> 
> ![image](https://user-images.githubusercontent.com/28700752/241936828-c060e0fd-7e68-4ac2-8656-226b0cd1bc51.png)
> 
> Iam pretty sure that's my name. that's also how it is issued on the official documents.

Ok.

**Reply by illyrius666** [reply]
> Is that your current course you are taking? Looks like too many on the page. Usually someone takes 4-6 classes.

the first image is the courses I am taking this semester. the other pages are all the courses I have to follow to get my degree. and yes its that much. that's what you get when you follow a degree that has architecture and engineering combined. this also takes 4 years to complete.

**Reply by jge162** [reply]
Ok try using the first image. To get approved.

**Reply by illyrius666** [reply]
Okay! i will try that!

**Reply by illyrius666** [reply, marked as answer]
That didnt work. but sending the receipt after 14x magically made it work. I am in now finally.

**Reply by CoolGenius-123** [reply]
Could you mentioned what documents you have submitted?

**Reply by illyrius666** [reply]
I used all kinds of documents ranging from site to payment receipt to the enrollment document I had to manually request from my uni. I eventually made it in with the payment receipt.

**Comment by SianaTDoe** [comment]
I have the same problem : I have my name, my bio, my school paper, all but I am rejected again and again, please git help...

**Reply by DimitarGL** [reply]
It's good to see somebody to have your problems :)
The same is to me-name is changed, my bio is added, screenshots are ok.... Replies still tell me to add my name and bio....
